src/System.IO.Abstractions.TestingHelpers/MockFileSystem.cs

```diff
@@ -124,6 +140,28 @@ private string GetPathWithCorrectDirectoryCapitalization(string fullPath)
             return fullPath.TrimSlashes();
         }
 
+        /// <inheritdoc />
+        public MockFileData AdjustTimes(MockFileData fileData, TimeAdjustments timeAdjustments)
+        {
+            var now = (dateTimeProvider ?? defaultDateTimeProvider)();
+            if ((timeAdjustments & TimeAdjustments.CreationTime) != TimeAdjustments.None)
+            {
+                fileData.CreationTime = now;
+            }
+
+            if ((timeAdjustments & TimeAdjustments.LastAccessTime) != TimeAdjustments.None)
+            {
+                fileData.LastAccessTime = now;
+            }
+
+            if ((timeAdjustments & TimeAdjustments.LastWriteTime) != TimeAdjustments.None)
```

Comment:
Same here 
```suggestion
            if ((timeAdjustments & TimeAdjustments.LastWriteTime) != TimeAdjustments.LastWriteTime)
```

Reply:
Will replace with `HasFlag` in next update